pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -34,6 +34,15 @@ def __lt__(self, requirement: 'Requirement') -> int:
         else:
             return self.name < requirement.name
 
+    def is_complete(self) -> bool:
+        return self.value is not None and not self.value.strip().endswith(b'\\')
+
+    def append_value(self, value: bytes):
```

Comment:
the linting is saying you need `-> None` on this function

Reply:
fixed